internal/commands/login/inputs.go

```diff
@@ -27,11 +30,18 @@ type inputs struct {
 	PrivateAPIKey string
 	Username      string
 	Password      string
+	NoBrowser     bool
 }
 
 func (i *inputs) Resolve(profile *user.Profile, ui terminal.UI) error {
 	u := profile.Credentials()
 
+	if u == (user.Credentials{}) && !i.NoBrowser {
+		if errBrowser := ui.OpenBrowser(apiKeysPage); errBrowser != nil {
+			_, _ = fmt.Fprintf(os.Stderr, "There was an issue opening your browser\n")
```

Comment:
[question] why are we calling `fmt.Fprintf` here?  I'd think our options were:

* return an error, let the `command_factory` handle printing back the message (and terminate command flow)
* use the `terminal.UI` to some debug/warning/error log but let the command continue

(note: there are reasons to choose any of the above approaches based on what behavior we actually want)

Reply:
Assuming we want the login to continue even if the browser doesn't work, I went with opt #2 --> using `terminal.UI `to output the error